app/templates/buyers/award.html

```diff
@@ -39,64 +39,62 @@
 {% block mainContent %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <div class="single-question-page">
-      <form method="POST" action="{{ url_for('.award_brief', framework_slug=brief.framework.slug, lot_slug=brief.lotSlug, brief_id=brief.id) }}">
-        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+    <form method="POST" action="{{ url_for('.award_brief', framework_slug=brief.framework.slug, lot_slug=brief.lotSlug, brief_id=brief.id) }}">
```

Comment:
Should we add `novalidate` to this form?

Reply:
Good call

Reply:
Pushed a commit adding novalidate to this and other forms